tests/configs/configs_test.go

```diff
@@ -85,4 +90,44 @@ var _ = Describe("Optional Configs", func() {
 			Expect(count).To(BeNumerically(">=", 1))
 		})
 	})
+	When("validating with kubeval", func() {
+		It("should pass", func() {
+			for _, v := range getSupportedK8Versions() {
+				By(fmt.Sprintf("checking version '%s'", v))
+				print("kubeval", "--strict", "--ignore-missing-schemas", "--skip-kinds", "Config", "-v", v, templatedPath)
```

Comment:
When I run this I see some missing whitespace (spaces and newlines)
```
STEP: checking version '1.18'
kubeval--strict--ignore-missing-schemas--skip-kindsConfig-v1.18/tmp/cf-for-k8s.ymlSTEP: checking version '1.14'
kubeval--strict--ignore-missing-schemas--skip-kindsConfig-v1.14/tmp/cf-for-k8s.yml
```

Reply:
I changed to use `command.args` which also prevents inconsistencies between the actual command and what we print.